### PR TITLE
Small changes noticed while reading "Machine management" book

### PR DESCRIPTION
**Pull request by jeana-redhat**
Applies to branches 4.6+ (with exceptions for any platforms that aren't supported in prior releases - will handle via cherrypick as appropriate).

- Fixed a few typos and other style issues.
- Fixed some assemblies that had the TOC in the wrong place.
- The list of assemblies in the `machine-user-provisioned-limitations.adoc` module was mostly incorrect. Removed a couple outdated lines, replaced one generic "Creating machine sets" reference with provider-specific refs and added the module to those assemblies.
- It appears Atom has also removed some extraneous spaces from the files I touched. 

Previews:
* [Creating a machine set on AWS (**IMPORTANT** note)](https://deploy-preview-35685--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-aws.html) - AWS as example, reused in other providers
* [Machine API overview](https://deploy-preview-35685--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws) - AWS as example, reused in other providers
* [Creating a machine set](https://deploy-preview-35685--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-aws.html#machineset-creating_creating-machineset-aws) - AWS as example, reused in other providers
* [Manually scaling a machine set](https://deploy-preview-35685--osdocs.netlify.app/openshift-enterprise/latest/machine_management/manually-scaling-machineset.html)
* [Creating infrastructure machine sets](https://deploy-preview-35685--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html)
* [Sample YAML for a machine set custom resource on Azure](https://deploy-preview-35685--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-azure.html#machineset-yaml-azure_creating-machineset-azure)
* [Enabling customer-managed encryption keys for a machine set](https://deploy-preview-35685--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-azure.html#machineset-enabling-customer-managed-encryption-azure_creating-machineset-azure)

### CODE REVIEW

**Review by JoelSpeed — COMMENTED** [review]
Based on my understanding of the current guidelines for product docs, all LGTM

File: modules/machine-user-provisioned-limitations.adoc [hunk 1]
```diff
@@ -1,13 +1,17 @@
 // Module included in the following assemblies:
 //
-// * machine_management/applying_autoscaling.adoc
 // * machine_management/creating-infrastructure-machinesets.adoc
-// * machine_management/creating-machinesets.adoc
```
Comment (jeana-redhat) [inline modules/machine-user-provisioned-limitations.adoc:5]: This assembly does not exist (and perhaps never did?). Replaced with provider-specific refs.

File: modules/machine-user-provisioned-limitations.adoc [hunk 2]
```diff
@@ -1,13 +1,17 @@
 // Module included in the following assemblies:
 //
-// * machine_management/applying_autoscaling.adoc
```
Comment (jeana-redhat) [inline modules/machine-user-provisioned-limitations.adoc:3]: This module was not referenced in the applying_autoscaling.adoc assembly (and does not seem to have ever been). The note covering similar material was [revamped in late 2019](https://github.com/openshift/openshift-docs/commit/fd0ab1e1875869e2f38189df8fef6baddfbabbf4#diff-1b7bfe0e4a765b714945f3db30721c9a6fb3d1b5a6cd11d76937701aecf85f5e). Removing this line reference.

File: modules/machine-user-provisioned-limitations.adoc [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 // * machine_management/creating-infrastructure-machinesets.adoc
```
Comment (jeana-redhat) [inline modules/machine-user-provisioned-limitations.adoc:3]: It wasn't actually in this assembly, but I have added it now.

**Review by JoelSpeed — COMMENTED** [review]
AFAIK, this looks correct

**Comment by jeana-redhat** [comment]
Hi @sunzhaohua2 and @miyadav -

I am not sure how much of the changed information here is testable/verifiable, but I would appreciate input from a QE perspective. Could one of you please let me know if you see any problems with what I have edited here?

Thanks!
Jeana

**Comment by miyadav** [comment]
Thanks @jeana-redhat , not sure if possible to fix style,because it was like this in previous version as well , api path now shown fully - example [/apis/machine.openshift.io/v1beta1/namespaces/{namespace}/machinesets/{n](https://deploy-preview-35685--osdocs.netlify.app/openshift-enterprise/latest/rest_api/machine_apis/machineset-machine-openshift-io-v1beta1.html#apismachine-openshift-iov1beta1namespacesnamespacemachinesetsname) , but this might be nitpicking .

overall LGTM .

**Comment by jeana-redhat** [comment]
Thanks @miyadav - apparently I should not have changed the API file at all (it's generated a different way), so I will revert that part in any case.

**Review by adellape — APPROVED** [review]
:fire:

**Pull request merged**